packages/app/pages/login.tsx

```diff
@@ -16,6 +26,38 @@ const Container = styled.div`
 const ContentContainer = styled.div`
   text-align: center;
 `;
+/**
+ * Opens a new window that directs a user to the Khoury OAuth login page and passes in
+ * the Office Hour client properties used for verification.
+ */
+function openKhouryOAuthLoginPage() {
+  let stateVal = "";
+  let codeVal = "";
+  if (isWindow) {
+    stateVal = window.localStorage.getItem("state");
+    codeVal = window.localStorage.getItem("challenge");
+  }
+  let md = forge.md.sha256.create();
+  md.update(codeVal);
+  const hashedCodeChallenge: string = md.digest().toHex();
+  const windowReference = window.open(
```

Comment:
Feel free to change it whenever we re-visit this, but can this be a string literal?